src/server.h

```diff
@@ -1789,6 +1817,7 @@ struct redisServer {
     int rdb_bgsave_scheduled;       /* BGSAVE when possible if true. */
     int rdb_child_type;             /* Type of save by active child. */
     int lastbgsave_status;          /* C_OK or C_ERR */
+    int primary_can_sync_using_rdb_channel; /* Track whether the primary is able to sync using rdb channel */
```

Comment:
too long. but also the term we use for now is "master", let's avoid causing further confusion and mix terms.
```suggestion
    int master_supports_rdb_channel; /* Track whether the master is able to sync using rdb channel */
```